detection-rules/attachment_pdf_link_to_zip_no_creator.yml

```diff
@@ -0,0 +1,17 @@
+name: "Attachment: PDF with single link to zip and creator source from /dev/shm"
+description: "Detects a PDF attachment with a single link to a ZIP file. Bumblebee delivery mechanism"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound 
+  and all(attachments, 
+    .file_extension == "pdf"
+    and any(file.explode(.), (
+      .depth == 0 
+      and length(filter(.scan.strings.strings, regex.contains(.,'\/S\/URI\/URI\(http(s):\/\/.+\.zip'))) == 1
```

Comment:
i'm guessing the URL scanner isn't pulling this out reliably?

Reply:
ScanUrl doesn't run on PDFs unfortunately. This is resolved in the new scanner, but I don't think we're ready to incorporate that yet.